### DISCUSSION TITLE
How to follow a VOR radial with AP

### DISCUSSION THREAD

**Discussion by lionhell** [flybywiresim/aircraft#5694] [Q&A]
Hi !
Is there a way to follow a VOR radial with AP ?
I tried to do that but did not find how to ?
Using the DIR button I can enter a VOR, but it seems that the "IN" and "OUT" buttons are inoperative !?
Impossible to enter a value... so the only way to use a VOR with DIR is to go straight to the station !
Am I right or do I miss something ?

THX

**Comment by tracernz** [comment]
Not implemented yet.
